Kim Edits (#50)
mulesoft/docs-partner-manager: modules/ROOT/pages/Certificates.adoc

> 

Reviewer: I will not be making organizational edits this close to publication, so...mostly copyediting for this round.

Author: Got it. Maybe for GA2?

> You can configure your SSL certificate settings for the different scenarios:

Reviewer: Not sure what "the different scenarios" are. There is no previous reference in this file, and what follows are different methods of configuring, so I am guessing that this should actually read...:
```suggestion
You can configure your SSL certificate settings by using one of the following methods:
```

Author: Rewrote with your edits in mind.

* Uploading your own, self-signed certificate
* Using a CA certificate (no upload is necessary)

> * Enable insecure connection for a test scenario

Reviewer: Parallelism:
```suggestion
* Enabling an insecure connection for a test scenario
```

Author: Fixed with my rewrite.

> For example, you may need your partner’s digital certificate such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol.

Reviewer: ```suggestion
For example, you might need your partner’s digital certificate, such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol.
```

Reviewer: Which method is the previous example referring to? What is it an example of: the need to configure settings or a method listed? I can't tell how this example relates to the previous list.

Reviewer: This table seems unnecessary to me, although not the information within it.
You already have a list of three ways to configure. Suggest putting the appropriate Description text below the list item to which it refers:
 
* Uploading your own self-signed certificate
   Upload a new certificate (.PEM format)...

* Using a CA certificate
   No upload necessary; the endpoint already uses a...

* Enabling an insecure connection for a test scenario
  Use for local testing....

Reviewer: Line 16
The table column heading Name is not accurate; none of what follows is a name or type

Reviewer: Line 18
Review style reference entry for "that" vs. "which" and validate that your sentence meaning requires "which," which should be preceded by a comma:

"Used for local testing, which doesn’t perform certificate validation."

Reviewer: Lines 20-21
"CA-signed certificate already in place; does not require certificate upload"

The two sentences used currently are redundant. Unless you intended the second sentence to be in the Description column, which it does not: formatting error?

Reviewer: Line 23
The "name" Certificates is not unique, or at least does not match the list item to which it refers, which is uploading your own self-signed certificate.

Reviewer: Line 25
Does this mean info that is included in the certificate or which the user must supply?

Reviewer: Lines 27-39
I would format these items as sentences. We do not ever want to use space-hyphen-space as a separator.

"`Authority`is the certificate provider.
`Serial Number` is the number of the certificate.
`Status` is green for active and red for inactive.
....

Author: Fixed.
